src/_data/capability-matrix.yml

```diff
@@ -202,6 +202,32 @@ categories:
             l1: 'Yes'
             l2: fully supported
             l3: ''
+      - name: Splittable DoFn
+        values:
+          - class: model
+            l1: 'Partially'
+            l2: DoFn with non-monolithic element processing
```

Comment:
Can we reverse the polarity here and describe `non-monolothic` instead? Something like "splitting and checkpointing of processing within an element"? My phrasing is not great, but something...